graphql/schema.graphql

```diff
@@ -81,4 +81,8 @@ type Mutation {
   setManualPrice(orderFormId: ID, input: ManualPriceInput!): OrderForm!
     @withOrderFormId
     @cacheControl(scope: PRIVATE)
+
+  itemsOrdination(ascending: Boolean!, criteria: String!): OrderForm!
```

Comment:
also, I think `updateItemsOrdination` is a better name for a mutation, but could be something like `setItemsOrdination` too
```suggestion
  updateItemsOrdination(ascending: Boolean!, criteria: String!): OrderForm!
```